packages/openfeature-web-provider/src/ConfidenceWebProvider.ts

```diff
@@ -16,6 +16,7 @@ import equal from 'fast-deep-equal';
 import { ApplyManager, ConfidenceClient, Configuration, ResolveContext } from '@spotify-confidence/client-http';
 
 const APPLY_TIMEOUT = 250;
+const MAX_APPLY_BUFFER_SIZE = 20;
```

Comment:
This needs to be discussed, I think this is an ok default. 

We should allow this to be set in the factory though

Reply:
This is the same batch size we chose for our mobile Providers. I think it's ok as default. Not sure it's worth to make this configurable by the users, as `apply` events are managed by us end-to-end

Reply:
happy to mirror the mobile sdks for this then :)